src/ziggurat/map.clj

```diff
@@ -0,0 +1,17 @@
+(ns ziggurat.map
```

Comment:
IMO this namespace should be in utils directory. as it has utils methods

Reply:
Namespace is changed